data/moves.js

```diff
@@ -7709,10 +7709,11 @@ exports.BattleMovedex = {
 		effect: {
 			onStart: function (target) {
 				this.add('-start', target, 'move: Leech Seed');
+				this.effectData.sourceSide = this.effectData.source.side;
 			},
 			onResidualOrder: 8,
 			onResidual: function (pokemon) {
-				var target = pokemon.side.foe.active[pokemon.volatiles['leechseed'].sourcePosition];
+				var target = this.effectData.sourceSide.active[pokemon.volatiles['leechseed'].sourcePosition];
```

Comment:
Why are you caching `sourceSide`? Can't you just do `.source.side` each time here?